packages/frontend/src/style.scss

```
@@ -35,7 +35,7 @@ html {
 	accent-color: var(--accent);
 	overflow: auto;
 	overflow-wrap: break-word;
-	font-family: 'Hiragino Maru Gothic Pro', "BIZ UDGothic", Roboto, HelveticaNeue, Arial, sans-serif;
+	font-family: Roboto, HelveticaNeue, Arial, sans-serif;
```

```suggestion
	font-family: Roboto, HelveticaNeue, 'Segoe UI Variable Text', 'Segoe UI', Arial, 'Hiragino Maru Gothic Pro', 'BIZ UDGothic', sans-serif;
```

Just change the priority and it should be fine (your original change makes it weird when displaying Japanese)

And I added some Windows-related fonts to make it look nicer

---

> See https://github.com/misskey-dev/misskey/issues/6858

そうなんだ